angular/lib/src/components/progressbar/progressbar.component.ts

```diff
@@ -1,20 +1,26 @@
-import type {ProgressbarContext, ProgressbarProps, ProgressbarWidget, SlotContent} from '@agnos-ui/angular-headless';
 import {
 	BaseWidgetDirective,
 	ComponentTemplate,
 	SlotDefaultDirective,
 	SlotDirective,
 	auBooleanAttribute,
 	auNumberAttribute,
-	callWidgetFactory,
-	createProgressbar,
 	useDirectiveForHost,
 } from '@agnos-ui/angular-headless';
+import type {AdaptWidgetSlots, WidgetProps, AdaptSlotContentProps, WidgetFactory, SlotContent} from '@agnos-ui/angular-headless';
 import {type WritableSignal, writable} from '@amadeus-it-group/tansu';
 import {NgClass} from '@angular/common';
 import type {AfterContentChecked} from '@angular/core';
 import {ChangeDetectionStrategy, Component, ContentChild, Directive, Input, TemplateRef, ViewChild, inject} from '@angular/core';
 
+import {createProgressbar as coreCreateProgressbar} from '@agnos-ui/style-bootstrap/components/progressbar';
+import {callWidgetFactory} from '../../config';
+
+type ProgressbarWidget = AdaptWidgetSlots<import('@agnos-ui/style-bootstrap/components/progressbar').ProgressbarWidget>;
+type ProgressbarProps = WidgetProps<ProgressbarWidget>;
+type ProgressbarContext = AdaptSlotContentProps<import('@agnos-ui/style-bootstrap/components/progressbar').ProgressbarContext>;
+const createProgressbar: WidgetFactory<ProgressbarWidget> = coreCreateProgressbar as any;
```

Comment:
Why `any`? shouldn't it be
```suggestion
const createProgressbar: WidgetFactory<ProgressbarWidget> = coreCreateProgressbar as WidgetFactory<ProgressbarWidget>;
```